src/Magpie.Core/OverlayDrawer.cpp

```diff
@@ -357,6 +357,8 @@ void OverlayDrawer::_BuildFontUI(std::wstring_view language, const std::vector<u
 		builder.AddRanges(fontAtlas.GetGlyphRangesCyrillic());
 	} else if (language == L"tr" || language == L"hu") {
 		builder.AddRanges(ImGuiHelper::Latin_1_Extended_A_RANGES);
+	} else if (language == L"vi") {
+		builder.AddRanges(fontAtlas.GetGlyphRangesVietnamese());
```

Comment:
This surprised me; I had intended to fix this in another commit. Great job!

Reply:
Yeah, caught it while checking everything, might as well fix it